tests/test_controller.py

```diff
@@ -125,7 +125,7 @@ async def test_task_connection(self):
             self.assertFalse(controller.are_clients_connected())
 
             # Wait a little time to reconstruct the connection
-            await asyncio.sleep(1)
+            await asyncio.sleep(2)
```

Comment:
Here and in the other sleep calls, consider making these sleep times into a constant so you don't have to updated them manually everywhere...

Reply:
Fixed in the commit: [449e880](https://github.com/lsst-ts/ts_m2com/pull/10/commits/449e8806e72c01fd253fbbdfc896198514aad657). Thanks!